pkg/minikube/assets/addons.go

```diff
@@ -389,7 +389,7 @@ var Addons = map[string]*Addon{
 			"0640"),
 	}, false, "registry", "Google", "", "", map[string]string{
 		"Registry":          "registry:2.8.1@sha256:83bb78d7b28f1ac99c68133af32c93e9a1c149bcd3cb6e683a3ee56e312f1c96",
-		"KubeRegistryProxy": "google_containers/kube-registry-proxy:0.4@sha256:1040f25a5273de0d72c54865a8efd47e3292de9fb8e5353e3fa76736b854f2da",
+		"KubeRegistryProxy": "k8s-minikube/kube-registry-proxy:0.0.1@sha256:498e7da3b171e24630615854c4a2924cfb8adbd43b0f252b1458c859ca9b746c",
```

Comment:
lets create an issue on Kubernetes and ask them where they plan to maintain the deleted code ?

Reply:
@BenTheElder mentioned anything under cluster has been depricated, so I think in minikube we should create an issue upstream to see if there is no other sponsor, minikube could host it on our repo

Reply:
Kubernetes/Kubernetes will not host this anymore since kube-up CI is not using it and excluding the etcd image this directory is only used for CI and not supported anymore (see cluster/README.md) but minikube could host these sources and iterate on it or SIG cluster lifecycle could sponsor bringing it back as a SIG project repo. I suspect the latter will not be supported though based on past conversations.

Reply:
SIG Testing + Infra is also trying to phase out the rest as well because we need to be running CI on multiple clouds instead of only the GCE bash. We're starting small with just one scale test on kops-aws but the hope is to use kOps AWS+GCE instead of kube-up GCE more generally, eventually. So there's not much future for anything in cluster/. It's barely maintained as-is out of necessity for CI.

Reply:
since there is no owner for this image, we could just publish one last patch of it in arm64 but in a follow up PR we could change the ownership of this addon from Google to minikube